bemani/utils/read.py

```diff
@@ -1318,10 +1329,18 @@ def __init__(
             VersionConstants.JUBEAT_PROP,
             VersionConstants.JUBEAT_QUBELL,
             VersionConstants.JUBEAT_CLAN,
+            VersionConstants.JUBEAT_FESTO,
+            VersionConstants.JUBEAT_PROP + DBConstants.OMNIMIX_VERSION_BUMP,
+            VersionConstants.JUBEAT_QUBELL + DBConstants.OMNIMIX_VERSION_BUMP,
+            VersionConstants.JUBEAT_CLAN + DBConstants.OMNIMIX_VERSION_BUMP,
+            VersionConstants.JUBEAT_FESTO + DBConstants.OMNIMIX_VERSION_BUMP,
         ]:
-            self.charts = [0, 1, 2]
+            self.charts = [0, 1, 2, 3, 4, 5]
```

Comment:
Or would it be better to change the number of charts here? That is, after all, why this lookup exists. We should be able to return 0-2 for everything but festo, and 0-5 for festo, and then the below stuff works.

Reply:
Another advantage of doing it this way is we can do what IIDX does: gray out charts that are impossible for certain songs (such as songs that aren't in festo having grayed out hard scores).

Reply:
I went ahead and made that change and pushed the update.